### DISCUSSION TITLE
3rd edition

### DISCUSSION THREAD

**Discussion by nsrmhjl** [nsrmhjl/ptDtSc#8] [Announcements]
<div dir='rtl'>
سلام آقای @AlirezaDoustmohammadi  
ادیشن سوم کتاب اومده 
</div>

[Tiago Antao - Bioinformatics with Python Cookbook_ Use modern Python libraries and applications to solve real-world computational biology problems, 3rd Edition-Packt Publishing (2022).pdf](https://github.com/nsrmhjl/ptDtSc/files/9725351/Tiago.Antao.-.Bioinformatics.with.Python.Cookbook_.Use.modern.Python.libraries.and.applications.to.solve.real-world.computational.biology.problems.3rd.Edition-Packt.Publishing.2022.pdf)

**Comment by AlirezaDostmohammadi** [comment]
<div dir="rtl">
سلام. خیلی ممنون دکتر جان
